src/Twig/EntryFilesTwigExtension.php

```diff
@@ -58,6 +60,19 @@ public function renderWebpackLinkTags(string $entryName, string $packageName = n
             ->renderWebpackLinkTags($entryName, $packageName, $entrypointName, $attributes);
     }
 
+    public function entryExists(string $entryName, string $entrypointName = '_default'): bool
+    {
+        try {
+            $entrypointLookup = $this->getEntrypointLookup($entrypointName);
+            $entrypointLookup->getJavaScriptFiles($entryName);
+            $entrypointLookup->reset();
```

Comment:
Hmm. I don't love this part.

Instead, what if we add an `entryExists()` method to `EntrypointLookup`. It would NOT be part of the interface - but a bonus method on that class.

Then, in here, we would have something like:

```php
$entrypointLookup = $this->getEntrypointLookup($entrypointName);
if (!$entrypointLookup instanceof EntrypointLookup) {
    throw new \LogicException(sprintf('Cannot use entryExists() unless the entrypoint lookup is an instance of "%s", EntrypointLookup::class));
}

return $entrypointLookup->entryExists($entryName);
```

This will avoid the "reset()" part, which could, in theory, have some side effects (if you have, somehow, already rendered some files from this entrypointlookup).